lfucg-exactions/server/accounts/helper.py

```diff
@@ -39,3 +42,42 @@ def send_lost_username_email(user):
     msg = EmailMultiAlternatives(subject, text_content, from_email, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+def send_email_to_supervisors(**kwargs):
+    if 'perm_name' in kwargs:
+        perm = Permission.objects.get(codename=kwargs['perm_name'])
+
+    q_objects = Q()
+    for group in kwargs['group']:
+        q_objects.add(Q(groups__name=group), Q.OR)
+    users = User.objects.filter(Q(profile__is_supervisor=True), q_objects)
+
+    to_emails = []
+    for user in users:
+        to_emails.append(user.email)
+
+    html_template = get_template('emails/supervisor_email.html')
+    text_template = get_template('emails/supervisor_email.txt')
+
+    subject = 'LFUCG Exactions Activity: Pending Approval - New ' + str(perm.content_type).title()
+    from_email = settings.DEFAULT_FROM_EMAIL
+
+    if 'entry' in kwargs:
+        entry_id = kwargs['entry']['id']
+
+    if 'pk' in kwargs:
+        entry_id = kwargs['pk']
+
+    context = {
+        'baseURL': settings.BASE_URL,
+        'model': perm.content_type,
+        'staticURL': 'https://lfucg-exactions-storage.s3.amazonaws.com/',
```

Comment:
this should be a variable in settings

Reply:
Wasn't working locally. I'd use settings.STATIC_URL otherwise.